testsuite/integration/elytron-oidc-client/src/test/java/org/wildfly/test/integration/elytron/oidc/client/OidcBaseTest.java

```diff
@@ -146,49 +148,49 @@ public static void checkDockerAvailability() {
     @Test
     @OperateOnDeployment(PROVIDER_URL_APP)
     public void testWrongPasswordWithProviderUrl() throws Exception {
-        loginToApp(PROVIDER_URL_APP, org.wildfly.test.integration.elytron.oidc.client.KeycloakConfiguration.ALICE, "WRONG_PASSWORD", HttpURLConnection.HTTP_OK, "Invalid username or password");
+        loginToApp(PROVIDER_URL_APP, org.wildfly.test.integration.elytron.oidc.client.KeycloakConfiguration.ALICE, "WRONG_PASSWORD", HttpURLConnection.HTTP_OK, "Invalid username or password", null, false);
```

Comment:
Instead of updating the existing references, we can introduce a new `loginToApp` method with the same parameters as before that just calls the new method with the additional values set to `null` and `false`.

Let me know if you have questions about that.